packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -53,6 +53,7 @@ export function ListToVirtualizedList(props) {
 			sortBy={sort && sort.field}
 			sortDirection={sort && sort.isDescending ? SORT_BY.DESC : SORT_BY.ASC}
 			type={props.displayMode.toUpperCase()}
+			noRowsRenderer={props.noRowsRenderer}
 			t={props.t}
```

Comment:
You can remove that, it no longer exists in VirtualizedList props.
Don't forget variable declaration and propTypes